### ISSUE TITLE
Masking output to make sure padded elements are not involved in computing loss function

### ISSUE THREAD

**Issue by nguyenvo09** [nguyenvo09/fake_news_detection_deep_learning#6]

**Comment by nguyenvo09** [comment]
Learning it from https://github.com/spro/practical-pytorch/blob/master/seq2seq-translation/seq2seq-translation-batched.ipynb
